styles.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+body {
+  background-image: url('images//background.png');
```

Comment:
- [ ] The background image should be inside the `section` that I mentioned above that you should change